moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -223,14 +222,15 @@ namespace DistanceRequestTypes
 {
 enum DistanceRequestType
 {
-  GLOBAL,   /// Find the global minimum
-  SINGLE,   /// Find the global minimum for each pair
-  LIMITED,  /// Find a limited(max_contacts_per_body) set of contacts for a given pair
-  ALL       /// Find all the contacts for a given pair
+  GLOBAL,   ///< Find the global minimum
+  SINGLE,   ///< Find the global minimum for each pair
+  LIMITED,  ///< Find a limited(max_contacts_per_body) set of contacts for a given pair
+  ALL       ///< Find all the contacts for a given pair
```

Comment:
Can you remind me what this formatting change is for?

Reply:
to get rid of the wrong table formatting [here](https://docs.ros.org/melodic/api/moveit_core/html/namespacecollision__detection_1_1DistanceRequestTypes.html)